include/oneapi/dpl/pstl/parallel_backend.h

```diff
@@ -35,6 +35,9 @@
 #    endif
 #endif
 
+//the parallel backend constants
+#define _ONEDPL_MERGE_CUT_OFF 2000
```

Comment:
Would it be better to make this a constexpr variable in an internal namespace?

Reply:
Actually, that constant was moved from `parallel_backend_tbb.h`....
I don't mind to do it as a `constexpr` variable.

Reply:
done.
`oneapi::dpl::__internal::__merge_algo_cut_off`